bower.json

```diff
@@ -34,7 +34,8 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#^1.3.0",
-    "PuppetJs": ">=1.3.8"
+    "PuppetJs": "puppetjs/puppetjs#feature/85-return-results-from-apply",
+    "underscore": "^1.8.3"
```

Comment:
I think it's not needed anymore, I used it at one point and then decided not to.